odh-manifests/osc-cl1/trino/base/trino-config-secret.yaml

```diff
@@ -179,6 +179,41 @@ stringData:
           { "name": "quant1", "allow": false }
         ]
       },
+      {
+        "group": "demo_dv_dev",
+        "catalog": "osc_datacommons_dev",
+        "schema": "demo",
+        "table": "gppd",
+        "privileges": ["SELECT", "INSERT", "DELETE", "OWNERSHIP"]
+      },
+      {
+        "group": "demo_dv_quant",
+        "catalog": "osc_datacommons_dev",
+        "schema": "demo",
+        "table": "gppd",
+        "privileges": ["SELECT"],
+        "filter": "country = \"France\""
```

Comment:
are double quotes in the query legal syntax?  I used single quotes, fwiw:
https://github.com/erikerlandson/apps/blob/f2506b26db693f570bb454b41de0a71093869a1a/odh-manifests/osc-cl1/trino/base/trino-config-secret.yaml#L169

Reply:
github is claiming this can mere with no rebase but you might want to rebase off latest from my pr just in case

Reply:
Corrected the quotes, hopefully it is correct now.